index.html

```diff
@@ -55,7 +55,7 @@ <h1 class="section-title">I'm Anderson Glad to see you!</h1>
     </section>
     <section id="works-section">
       <div class="work-container">
-        <img class="align-item" src="./assets/images/work-1.png" alt="Tonic">
+        <img class="align-item card-image" src="./assets/images/work-1.png" alt="Tonic">
         <div class="work-aditional-info">
           <div class="work-info-container">
```

Comment:
- [ ] I think there is a `50%`, `50%` split between the left and right sides of your about section. Kindly update it so the left side is occupying `40%` and the right side is taking the remaining `60%` of the parent container. 🤝️

- **Your Design**

![Screenshot from 2022-12-10 08-06-00](https://user-images.githubusercontent.com/17243325/206831330-c3a62156-60e1-4580-90e1-3516ed5829e0.png)